includes/Controllers/Admin/WidgetController.php

```diff
@@ -45,7 +45,7 @@ public function createWalletCharge( WP_REST_Request $request ) {
 				$image = wp_get_attachment_image_url( get_post_thumbnail_id( $item['id'] ), 'full' );
 
 				$itemData = [
-					'amount'   => $item['prices']['price'],
+					'amount'   => round( $item['prices']['price'] / 100, 2 ),
```

Comment:
What is the purpose of the new calculation? What is it fixing.

Reply:
after debugging, I've detected how we use the WooCommerce API.

under the:
class ProductSchema
protected function prepare_product_price_response

and:
abstract class AbstractSchema
protected function prepare_money_response

it returns our values as shown in the image below:
https://prnt.sc/c_taEco--I3Q

these prices are $15, $20, and $30 at checkout.

previously, I mistakenly removed the "price / 100" logic, which caused a PayPal bug, changing $12 to $1200.

this issue has now been resolved